features/luks/data/lib/uevent/handlers/085-luks

```diff
@@ -176,11 +176,24 @@ cryptsetup_ask_pass() {
 	} > /.initrd/cryptsetup-ask-pass
 
 	if plymouth --ping 2>/dev/null; then
-		plymouth ask-for-password \
-			--prompt "Please enter passphrase for $LUKS_ROOT:" \
-			--number-of-tries=$tries \
-			--command="bash /.initrd/cryptsetup-ask-pass"
-		rc="$?"
+		if unl0kr -V 2>/dev/null; then
+			plymouth hide-splash 2>/dev/null
+				while [ $tries -ne 0 ]; do
```

Comment:
This indentation between two plymouth show/hide-splash actions that logically linked. Should I shift it?

Reply:
I've did it better and add luks-unl0kr for smart-cards. Maybe the password request should be more generalized, for example by adding a new ask_pass() function?

Reply:
@Vedingrot I've already started thinking about it. I think that in the future it will be necessary to create a separate common function or utility for requesting a password.